src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -83,25 +102,45 @@ const ExecutiveEducation2UPage = () => {
       {!queryParams.failureReason && (
         <>
           <h2 className="mb-3">
-            {isLoading ? (
+            {isLoading || !contentMetadata ? (
               <Skeleton containerTestId="loading-skeleton-page-title" />
-            ) : pageTitle}
+            ) : 'Your Registration(s)'}
```

Comment:
nit: casing is off; should be "Your registration(s)": 

<img width="247" alt="image" src="https://user-images.githubusercontent.com/2828721/231522258-e3b6d7a6-c8b0-4db5-908c-3f3d7eecdd8c.png">